packages/components/src/tabs/types.ts

```diff
@@ -26,12 +26,10 @@ export type TabsProps = {
 	/**
 	 * Determines if the tab should be selected when it receives focus. If set to
 	 * `false`, the tab will only be selected upon clicking, not when using arrow
-	 * keys to shift focus (manual tab activation). See the official W3C docs
+	 * keys to shift focus (manual tab activation). See the [official W3C docs](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/)
 	 * for more info.
 	 *
 	 * @default true
-	 *
-	 * @see https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/
```

Comment:
We should avoid `@see` tags as they are suppressed in Storybook props tables, and don't make sense when displayed unformatted in the auto-generated README.